packages/interface/src/components/complementary-area/style.scss

```diff
@@ -1,3 +1,4 @@
+.interface-interface-skeleton__actions,
```

Comment:
So this is obviously a big change, given it's SCSS. But at the moment, it targets the entire inspector, why shouldn't it target also the pre-publish inspector? Would love a sanity check on this one. Thank you!

Reply:
Just speaking from a general CSS architecture standpoint, this kind of thing isn't encouraged because it breaks component encapsulation. Styles for `InterfaceSkeleton` should not live in a different component's folder.

https://github.com/WordPress/gutenberg/blob/db3b6478ae8254252c9848094bb88761a963bc85/packages/interface/src/components/complementary-area/style.scss#L44-L51

And assuming that the reason you want to do this is to get these uppercase styles ☝️, this kind of styling (targeting all `h3`s) also isn't generally tenable in a componentized app. 

So my quick fix suggestion for this specific issue would be to change this:

https://github.com/WordPress/gutenberg/blob/db3b6478ae8254252c9848094bb88761a963bc85/packages/editor/src/components/post-taxonomies/most-used-terms.js#L50-L52

to:

```jsx
<BaseControl.VisualLabel as="h3" className="editor-post-taxonomies__flat-term-most-used-label"> 
  { label } 
</BaseControl.VisualLabel> 
```

so we can get the label styles cleanly. Let me know if you need further assistance, and I can push a commit to this branch if you would like.

Reply:
Please feel free to push to this branch!

Thank you.